src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/jpa/entity/RecommendationEntity.kt

```diff
@@ -32,17 +33,27 @@ data class RecommendationEntity(
 data class RecommendationModel(
   val crn: String?,
   var recallType: RecallType? = null,
+  var custodyStatus: CustodyStatus? = null,
   var status: Status? = null,
   var lastModifiedBy: String? = null,
   var lastModifiedDate: String? = null,
   val createdBy: String? = null,
   val createdDate: String? = null
 ) : Serializable
 
+enum class Recommendation(val text: String) {
+  STANDARD("Standard"), NO_RECALL("No recall"), FIXED_TERM("Fixed term"),
```

Comment:
I don't know if this impacts you but the labels are going to change slightly in the front end - `Standard recall`, `Fixed term recall`, `No recall`.
Should they be hardcoded here? The source of truth is the UI

Reply:
These aren't needed anymore :)
Let me check out your branch Jack to see if it all still works with these removed.

Reply:
Actually, that's a rebase issue - this shouldn't be in there, Bill removed as part of his last PR. I'll fix now...

Reply:
ah okay :)

Reply:
👍